mec15xxevb_assy6853:crypto.tinycrypt.hmac_prng.hmac_prng faild to build,

Describe the bug
building tests/crypto/tinycrypt_hmac_prng/ on mec15xxevb_assy6853 board, it would report a build error.It showed me SRAM OVERFLOW

To Reproduce
Steps to reproduce the behavior:

     west build -b mec15xxevb_assy6853 -p auto tests/crypto/tinycrypt_hmac_prng/
    see error

Logs and console output
[138/143] Linking C executable zephyr/zephyr_prebuilt.elf
FAILED: zephyr/zephyr_prebuilt.elf 
: && ccache /home/ztest/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc    zephyr/CMakeFiles/zephyr_prebuilt.dir/misc/empty_file.c.obj  -o zephyr/zephyr_prebuilt.elf  -Wl,-T zephyr/linker.cmd -Wl,-Map=/home/ztest/work/zephyrproject/zephyr/build/zephyr/zephyr_prebuilt.map -Wl,--whole-archive app/libapp.a zephyr/libzephyr.a zephyr/arch/common/libarch__common.a zephyr/arch/arch/arm/core/aarch32/libarch__arm__core__aarch32.a zephyr/arch/arch/arm/core/aarch32/cortex_m/libarch__arm__core__aarch32__cortex_m.a zephyr/lib/libc/minimal/liblib__libc__minimal.a zephyr/lib/posix/liblib__posix.a zephyr/boards/arm/mec15xxevb_assy6853/libboards__arm__mec15xxevb_assy6853.a zephyr/subsys/testsuite/ztest/libsubsys__testsuite__ztest.a zephyr/drivers/adc/libdrivers__adc.a zephyr/drivers/gpio/libdrivers__gpio.a zephyr/drivers/i2c/libdrivers__i2c.a zephyr/drivers/pwm/libdrivers__pwm.a zephyr/drivers/sensor/mchp_tach_xec/libdrivers__sensor__mchp_tach_xec.a zephyr/drivers/espi/libdrivers__espi.a zephyr/drivers/ps2/libdrivers__ps2.a zephyr/drivers/serial/libdrivers__serial.a -Wl,--no-whole-archive zephyr/kernel/libkernel.a zephyr/CMakeFiles/offsets.dir/arch/arm/core/offsets/offsets.c.obj -L"/home/ztest/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/thumb/v7e-m/nofp" -L/home/ztest/work/zephyrproject/zephyr/build/zephyr -lgcc -Wl,--print-memory-usage -mcpu=cortex-m4 -mthumb -mabi=aapcs -Wl,--gc-sections -Wl,--build-id=none -Wl,--sort-common=descending -Wl,--sort-section=alignment -Wl,-u,_OffsetAbsSyms -Wl,-u,_ConfigAbsSyms -nostdlib -static -no-pie -Wl,-X -Wl,-N -Wl,--orphan-handling=warn && :
Memory region         Used Size  Region Size  %age Used
           FLASH:       88124 B       224 KB     38.42%
            SRAM:       46872 B        32 KB    143.04%
        IDT_LIST:         216 B         2 KB     10.55/home/ztest/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: **zephyr/zephyr_prebuilt.elf section `noinit' will not fit in region `SRAM'**
/home/ztest/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: section .intList VMA [0000000000120000,00000000001200d7] overlaps section noinit VMA [0000000000118c18,0000000000123717]
/home/ztest/zephyr-sdk-0.11.4/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/../../../../arm-zephyr-eabi/bin/ld: **region `SRAM' overflowed by 14104 bytes**
collect2: error: ld returned 1 exit status


Environment (please complete the following information):

    OS: Fedora28
    Toolchain: Zephyr-sdk-0.11.3
    Commit ID: 34859d0d3e

--- a/tests/crypto/tinycrypt_hmac_prng/testcase.yaml
+++ b/tests/crypto/tinycrypt_hmac_prng/testcase.yaml
@@ -1,5 +1,6 @@
 tests:
   crypto.tinycrypt.hmac_prng:
     tags: tinycrypt crypto sha256 hmac prng
+    platform_exclude: mec15xxevb_assy6853 mec1501modular_assy6885
     integration_platforms:
       - native_posix

